File: prime_factors.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(int argc, char const *argv[])
{
  long int n = 745358336;
  int factors[78];
  int i = 3, j = 0;
  while(n % 2 == 0) {
    cout << 2 << " " << endl;
    factors[j++] = 2;
    n /= 2;
  }
  double max_factor = sqrt(n);
  while(i < max_factor) {
    while(n % i == 0) {
      n /= i;
      j++;
      // factors[j++] = i;
      cout << j << " " << i << endl;
      max_factor = sqrt(n);
    }
    i++;
  }
  if(n > 1) factors[j] = n;  
  return 0;
}

File: prime_factors.py
"""
  program for finding prime factors of a given number
  - Bhanu Prakash P
    15/5/2016
"""
import math
n = 55686132
factors = []
i = 3
while n % 2 == 0:
  n /= 2
  factors.append(2)
max_factor = math.floor(math.sqrt(n))
while(i < max_factor):
  while(n % i == 0):
    factors.append(i)
    n /= i
    max_factor = math.floor(math.sqrt(n))
  i += 1
if n > 1: factors.append(n);
print factors    